src/main/java/com/hollingsworth/arsnouveau/common/spell/effect/EffectFell.java

```diff
@@ -43,14 +43,14 @@ public void onResolveBlock(BlockHitResult ray, Level world, @NotNull LivingEntit
                     return;
                 if (spellStats.hasBuff(AugmentExtract.INSTANCE)) {
                     world.getBlockState(listPos).getDrops(LootUtil.getSilkContext((ServerLevel) world, listPos, shooter)).forEach(i -> world.addFreshEntity(new ItemEntity(world, listPos.getX(), listPos.getY(), listPos.getZ(), i)));
-                    BlockUtil.destroyBlockSafelyWithoutSound(world, listPos, false);
+                    BlockUtil.destroyBlockSafelyWithoutSound(world, listPos, false, shooter);
                 } else if (spellStats.hasBuff(AugmentFortune.INSTANCE)) {
                     world.getBlockState(listPos)
                             .getDrops(LootUtil.getFortuneContext((ServerLevel) world, listPos, shooter, spellStats.getBuffCount(AugmentFortune.INSTANCE)))
                             .forEach(i -> world.addFreshEntity(new ItemEntity(world, listPos.getX(), listPos.getY(), listPos.getZ(), i)));
-                    BlockUtil.destroyBlockSafelyWithoutSound(world, listPos, false);
+                    BlockUtil.destroyBlockSafelyWithoutSound(world, listPos, false, shooter);
```

Comment:
This is going to cause dupes as the above code is making drops without a promise the block was destroyed. We have to check that the block was successfully destroyed first.